studio/hooks/misc/useStore.tsx

```diff
@@ -40,7 +40,12 @@ export const StoreProvider: FC<StoreProvider> = ({ children, rootStore }) => {
 
   useEffect(() => {
     ui.load()
-    window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', matchMediaEvent)
+
+    if (window.matchMedia('(prefers-color-scheme: dark)').addEventListener) {
```

Comment:
```suggestion
    if (window?.matchMedia('(prefers-color-scheme: dark)')?.addEventListener) {
```

Not sure if this is necessary - could require `?` since `window.matchMedia('(prefers-color-scheme: dark)')`  might return `undefined`?

Reply:
It looks like window.matchMedia won't return undefined.
lets add `?` anyway, won't hurt